samples/automl/automlVisionPredict.js

```diff
@@ -0,0 +1,136 @@
+/**
+ * Copyright 2018, Google, LLC.
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * This application demonstrates how to perform basic operations on dataset
+ * with the Google AutoML Natural Language API.
+ *
+ * For more information, see the documentation at
+ * https://cloud.google.com/natural-language/automl/docs/
+ */
+
+'use strict';
+
+function predict(projectId, computeRegion, modelId, filePath, scoreThreshold) {
+  // [START automl_vision_predict]
+  const automl = require('@google-cloud/automl');
+  const fs = require('fs');
+
+  // Create client for prediction service.
+  const client = new automl.v1beta1.PredictionServiceClient();
+
+  /**
+   * TODO(developer): Uncomment the following line before running the sample.
+   */
+  // const projectId = `The GCLOUD_PROJECT string, e.g. "my-gcloud-project"`;
+  // const computeRegion = `region-name, e.g. "us-central1"`;
+  // const modelId = `id of the model, e.g. “ICN12345”`;
+  // const filePath = `local text file path of content to be classified, e.g. "./resources/test.txt"`;
+  // const scoreThreshold = `value between 0.0 and 1.0, e.g. "0.5"';
+
+  // Get the full path of the model.
+  const modelFullId = client.modelPath(projectId, computeRegion, modelId);
+
+  // Read the file content for prediction.
+  const content = fs.readFileSync(filePath, 'base64');
+
+  var params = {};
+  if (scoreThreshold) {
+    params = {
+      scoreThreshold: scoreThreshold,
+    };
+  }
+
+  // Set the payload by giving the content and type of the file.
+  const payload = {
+    image: {
+      imageBytes: content,
+    },
+  };

```

Comment:
1) As mentioned elsewhere - remove the `var` above in favor of `const` or `let
2) Can we shorten this like so?
```javascript
const params = {}
if (scoreThreshold} {
  params.scoreThreshold = scoreThreshold;
}
```